docs/getting-started/server/public-api.md

```diff
@@ -0,0 +1,59 @@
+# Public API
+
+The Bitwarden Public API provides organizations a suite of tools for managing members, collections,
+groups, event logs, and policies. More information about the Public API is available in the
+[Help Center](https://bitwarden.com/help/public-api/).
+
+## Differences with the private API
+
+Most developers will be more familiar with the private API used by our client applications. The
+Public API is different in several key areas:
+
+<!-- prettier-ignore -->
+| Private API | Public API |
+| --- | --- |
+| Located at api.bitwarden.com | Located at api.bitwarden.com/public |
+| Used by official Bitwarden client applications | Used by third parties, usually in custom integrations |
+| Broad scope - can be used for anything | Narrow scope - can only be used to manage organizations |
+| Can be changed without notice | Must give notice for any breaking changes
```

Comment:
⚠️ I disagree with the phrasing here -- we plan with breaking changes. Perhaps this can just drop the "breaking changes" and the below elaborate on what breaking changes means.

Pointing out https://bitwarden.com/help/bitwarden-software-release-support/ with the notice and windowing could be a benefit too.

Reply:
By "notice" I meant "notice to third parties/customers". We can and do change the internal api all the time without expressly telling customers. cf. publishing deprecation notices/warnings.

I've tweaked the wording here and included that link. Happy to adjust it further if required, but in that case please provide some suggested wording.